docs/source/conf.py

```diff
@@ -17,7 +17,7 @@
 sys.path.insert(0, os.path.abspath(os.path.join("..", "..")))
 
 try:
-    import numpydoc
+    pass
```

Comment:
Question: Was this an intended change or an auto-fix? This looks like an odd pattern

Reply:
Oh yikes - this was an autofix. I'm going to revert it, because I don't remember why I had the guarded numpydoc import to being with...